pkg/mumbleproto/Makefile

```diff
@@ -0,0 +1,6 @@
+Mumble.proto:
```

Comment:
I would prefer it if this used //go:generate instead.

For example:

generate.go:

    // -build
    //go:generate go run sync.go

sync.go:

    // -build
   func main() {
       // fetch file
       // fix package name
       // write to disk
   }

Then it would also work on Windows. (and OS X/BSD, where your sed change unfortunately wouldn't work: http://unix.stackexchange.com/questions/92895/how-to-achieve-portability-with-sed-i-in-place-editing)